File: SDKs/amazonaws.com/ec2/2016-11-15/typescript/src/sdk/models/shared/describeelasticgpusresult.ts
import { SpeakeasyMetadata, SpeakeasyBase } from "../../../internal/utils";



export class DescribeElasticGpusResult extends SpeakeasyBase {
  @SpeakeasyMetadata()
  elasticGpuSet?: Record<string, any>;

  @SpeakeasyMetadata()
  maxResults?: Record<string, any>;

  @SpeakeasyMetadata()
  nextToken?: Record<string, any>;
}

File: SDKs/amazonaws.com/ec2/2016-11-15/python/src/sdk/models/shared/describeelasticgpusresult.py
import dataclasses
from typing import Any,Optional


@dataclasses.dataclass
class DescribeElasticGpusResult:
    elastic_gpu_set: Optional[dict[str, Any]] = dataclasses.field(default=None)
    max_results: Optional[dict[str, Any]] = dataclasses.field(default=None)
    next_token: Optional[dict[str, Any]] = dataclasses.field(default=None)
    
